### ISSUE TITLE
investigate using node-notifier package

### ISSUE THREAD

**Issue by oakmac** [oakmac/cuttle#69]
for desktop notifications

**Comment by shaunlebron** [comment]
Nice find, just read through it and it looks like it tries all possible native solutions before falling back to Growl.  That would keep us from having to include Growl in the install. would take care of #67 and #68

**Comment by CASandmann** [comment]
Yeah, the application is already 100MB unzipped, I would prefer if we didn't make the user install any more stuff on top of that.
